### PR TITLE
2271 - Fix button inside the popover

### LINKED ISSUE #1576: Popover: X button alignment issue in iOS and Windows IE
**Describe the bug**
Popover: X button alignment issue in iOS and Windows IE

**To Reproduce**
1. Open Windows IE and iPhone X Safari
2. Go to https://4150-beta0-enterprise.demo.design.infor.com/components/popover/example-header-color.html
3. Open the Popover
4. See issue: X button is not aligned correctly

**Expected behavior**
X button in Popovers should be in the correct position

**Version**
- ids-enterprise: v4.14.0 to v4.15.0-beta0

**Screenshots**
Windows IE
screen shot 2019-01-31 at 4 16 41 pm

iPhone X Safari
screen shot 2019-01-31 at 4 18 09 pm

**Platform**
 - Windows Internet Explorer
 - iPhone Safari

**Additional context**
No additional context.

### LINKED ISSUE #2271: Popover: Button overflows container
**Describe the bug**
The button inside the popover overflows it's container at small screen sizes.

**To Reproduce**
- Run demo app
- Visit http://localhost:4000/components/popover/example-alternate-positions.html or http://localhost:4000/components/popover/example-index.html 
- Reduce screen size to mobile
- Click either button
  - notice the "Apply" button overflows it's container

**Expected behavior**
I expect the popover to be contained to the dimensions of the device and for the button to not overflow it's container.

**Version**
v4.20.0

**Screenshots**
bug

**Platform**
 - iPhone 5/SE (simulated in Dev Tools on Chrome Desktop)
 - MacOS 10.14.5
 - Chrome Version 74.0.3729.169 (Official Build) (64-bit)

**Additional Context**
First reported here: https://github.com/infor-design/enterprise/pull/2267#issuecomment-497118814

### PR DESCRIPTION
**Pull request by deep7102**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Fixed button inside the popover was overflowing from container on mobile screen size and the x-button was not aligning.

**Related github/jira issue (required)**:
Closes #1576
Closes #2271

**Steps necessary to review your pull request (required)**:
http://localhost:4000/components/popover/example-alternate-positions.html
http://localhost:4000/components/popover/example-index.html
- Pull this branch and build/run the demo app
- Using iPhone 5/SE or resize browser window to smaller mobile screen size
- Visit above links
- Click button to open popover
- See `Apply` button should not overflow it's container

http://localhost:4000/components/popover/example-header-color.html
- Open above link with windows ie11 or iPhone X safari
- Click button to open popover
- See X-button should not be misalign

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

**Comment by EdwardCoyle** [comment]
The (failing e2e test)[https://travis-ci.com/infor-design/enterprise/jobs/212761489#L1233] is unrelated to the changes in this PR... looks like it's [the new one from your PR](https://github.com/infor-design/enterprise/pull/2459/files#diff-871806765f6dadb98fa4b85fc3bfec4aR1253) @tmcconechy ?

**Comment by tmcconechy** [comment]
Maybe slow opening the lookup and need a sleep on that test. I can add tomorrow unless you can add?

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -55,6 +55,8 @@
 - `[Modal]` Fixed an issue where the modal component would disappear if its content had a checkbox in it in RTL. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))
 - `[Personalization]` Updated some of the colors to more readable in contrast mode. ([#2097](https://github.com/infor-design/enterprise/issues/2097))
 - `[Popdown]` Fixed usability issue where the Popdown could close prematurely when attempting to use inner components, such as Dropdowns. ([#2092](https://github.com/infor-design/enterprise/issues/2092))
+- `[Popover]` Fixed an issue where x-button was not aligning. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md]:
```suggestion
- `[Popover]` Fixed an issue where close button was not aligning. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
```

**Comment by deep7102** [comment]
Hi @tmcconechy updated now shorter header

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -60,6 +60,8 @@
 - `[Modal]` Fixed an issue where the modal component would disappear if its content had a checkbox in it in RTL. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))
 - `[Personalization]` Updated some of the colors to more readable in contrast mode. ([#2097](https://github.com/infor-design/enterprise/issues/2097))
 - `[Popdown]` Fixed usability issue where the Popdown could close prematurely when attempting to use inner components, such as Dropdowns. ([#2092](https://github.com/infor-design/enterprise/issues/2092))
+- `[Popover]` Fixed an issue where close button was not aligning. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md]:
```suggestion
- `[Popover]` Fixed an issue where close button was not aligned correctly. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
```

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -60,6 +60,8 @@
...
+- `[Popover]` Fixed an issue where inside button was overflowing from container on mobile screen size. ([#2271](https://github.com/infor-design/enterprise/issues/2271))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md]:
```suggestion
- `[Popover]` Fixed an issue where buttons inside the popover would overflow this popover container on smaller screen size. ([#2271](https://github.com/infor-design/enterprise/issues/2271))
```

File: docs/CHANGELOG.md [hunk 4]
```diff
@@ -60,6 +60,8 @@
 - `[Modal]` Fixed an issue where the modal component would disappear if its content had a checkbox in it in RTL. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))
 - `[Personalization]` Updated some of the colors to more readable in contrast mode. ([#2097](https://github.com/infor-design/enterprise/issues/2097))
 - `[Popdown]` Fixed usability issue where the Popdown could close prematurely when attempting to use inner components, such as Dropdowns. ([#2092](https://github.com/infor-design/enterprise/issues/2092))
+- `[Popover]` Fixed an issue where close button was not aligned correctly. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
+- `[Popover]` Fixed an issue where buttons inside the popover would overflow this popover container on smaller screen size. ([#2271](https://github.com/infor-design/enterprise/issues/2271))
```
Comment (davidcarlsonberg) [inline docs/CHANGELOG.md]:
```suggestion
- `[Popover]` Fixed an issue where buttons inside the popover would overflow at smaller screen sizes. ([#2271](https://github.com/infor-design/enterprise/issues/2271))
```

File: docs/CHANGELOG.md [hunk 5]
```diff
@@ -60,6 +60,8 @@
 - `[Modal]` Fixed an issue where the modal component would disappear if its content had a checkbox in it in RTL. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))
 - `[Personalization]` Updated some of the colors to more readable in contrast mode. ([#2097](https://github.com/infor-design/enterprise/issues/2097))
 - `[Popdown]` Fixed usability issue where the Popdown could close prematurely when attempting to use inner components, such as Dropdowns. ([#2092](https://github.com/infor-design/enterprise/issues/2092))
+- `[Popover]` Fixed an issue where close button was not aligned correctly. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
```
Comment (davidcarlsonberg) [inline docs/CHANGELOG.md]:
```suggestion
- `[Popover]` Correctly align the close button in a popover at smaller screen sizes. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
```

**Pull request merged**